www/scipylib/building/windows.rst

```diff
@@ -32,185 +32,257 @@ information than this document will provide.
+
+Building the Released SciPy
+---------------------------
+
+This section provides the step-by-step process to build the released scipy. If you want
+to build completely from source, you should estimate at least three hours to build all
+libraries and compile SciPy. Feel free to stop and inspect any step at any time, but
+for this section, we'll just mention the steps without providing an in-depth explanation
+for the reasons behind them. If you have further questions about what we're doing, more
+in-depth documentation is provided in the sections below. Also, please make sure to read
+this section before proceeding, as the presence or absence of error messages in general
+is not a good indication of whether you've completed a step correctly. Each step creates
+particular files, and what ultimately matters is whether you have built the required files
+rather than whether error messages appeared in your terminal.
+
+Building OpenBLAS
+=================
+
+First, we need to install the software required to build OpenBLAS_, which is the BLAS_
+library that we're going to use. Because the software to build OpenBLAS is different than
+that required to build SciPy and because OpenBLAS takes a long time to build, we're going
+to start building OpenBLAS first and then explain what to do next while the OpenBLAS build
+is running. **Alternatively, if you'd rather download a pre-built OpenBLAS, download the**
+**one of the** `pre-built zip files`_ **and skip to the Installing OpenBLAS section below.**
+
+Otherwise, install MSYS2 using `these instructions`_ including the `pacman` update
+instructions. Occasionally during the updates the terminal might ask you to close the
+terminal but refuses to be closed and hang. If this happens you can kill it via Task Manager
+and continue with the instructions. Make sure to install the correct architecture for the
+SciPy that you want to build (eg. 32 or 64 bit). Now, you have three options for opening a
+terminal which are MSYS2, MINGW (32bit,64bit). After updating all the packages, now we are 
+ready to install some more package bundles that we will need. Open a MSYS2 terminal 
+and type the following depending on the architecture of your choice:
+
+.. code:: shell
+
+    pacman -S --needed base-devel mingw-w64-i686-toolchain mingw-w64-i686-cmake git
+
+for 32-bit and 
+
+.. code:: shell
+
+    pacman -S --needed base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake git
+
+for 64-bit build. It will prompt to whether install everything in these packages and 
+you can simply accept all via hitting enter key at each step. 
+
+We should be aware of the fact that these tools also install Python2, very similar to 
+a virtual environment, which is only usable within MSYS2 terminals and we are 
+**not** going to use it at any point. Therefore, after the OpenBLAS build we are 
```

Comment:
Based on the recent changes, I think this sentence is no longer true as we keep on building scipy within MSYS2 right?